packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -78,12 +78,43 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
 
 <sd-accordion>
   <h3 className="text-base sb-unstyled" slot="summary">
-    {' '}
-    Which browsers are supported?
+    What is the suggested workflow between Design and DEV in the project-specific teams? 
   </h3>
   <span className="sb-unstyled">
-    Currently the we provide auomatic VRT in Chrome and manually test our components with MS Edge, Safari and Firefox. A
-    more detailed browser matrix is in progress and will be added here soon.
+    A collaborative workflow between design and development teams is crucial for the success of any project. We recommend the following steps:
+
+    1. Initial Alignment: Begin with a kick-off meeting to ensure both designers and developers understand Solid DS and its capabilities.
+    2. Design Phase: Designers build prototypes based on Solid DS components, customising them whenever necessary while collaborating with developers to ensure feasibility. They also make all the relevant accessibility comments for later hand-off.
+    3. Hand-off Process: Designers provide detailed documentation and conduct design review sessions to ensure developers understand the specifications and rationale.
+    4. Development Phase: Maintain close communication, with regular check-ins and iterative testing to ensure designs are implemented accurately.
```

Comment:
This is definitely not my recommendation and feels way to waterfall-ish. Why do we have to do this info anyways for other teams here?

Reply:
This question came out of the FAQ-survey we did on our General channel. Similarly to the question about the core principles of the Solid DS, this is also a way of showcasing our principles rather than answering a real question IMO. 

I agree though that the current answer feels waterfall-ish. I would suggest the following text, which is more mindset-oriented rather than prescribing a particular workflow:

> The Solid Design System team does not impose strict rules on how design and development teams should operate. Instead, we offer guidance and best practices that promote collaboration, agility, and efficiency in integrating the design system into your workflow. The collaboration advice for consumers generally includes:
> 
> Shared Language and Terminology
> Ensuring that both designers and developers refer to components, colors, or tokens using a unified language helps avoid confusion and miscommunication during implementation.
> 
> Use of Agile Practices
> Consumers are encouraged to adopt agile workflows, such as frequent design and development sprints. This allows for continuous feedback and adjustments to ensure that both the design and development stay aligned​.
> 
> Design Systems as a Source of Truth
> The design system should serve as the central repository for all reusable components, ensuring consistency across teams and reducing the need for repetitive design work​.